extension/package.json

```diff
@@ -34,7 +35,8 @@
     "npm-audit-resolver": "^1.4.0",
     "nyc": "^13.1.0",
     "rewire": "^2.5.2",
-    "underscore": "1.8.3"
+    "underscore": "1.8.3",
```

Comment:
is it needed?

Reply:
yes - for example in deleteItemsFromCart.js

Reply:
seems like the only usage, can you just quickly refactor that _.each? We can just do something like this:
```
input.couponCodes.forEach(couponCode => {
      cartItemIds.push('COUPON_' + couponCode)
    })
```